Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -676,6 +676,12 @@ void SourceBufferPrivate::didReceiveInitializationSegment(InitializationSegment&
     m_pendingOperations.append({ WTFMove(initOperation) });
 }
 
+void SourceBufferPrivate::didUpdateFormatDescriptionForTrackId(Ref<TrackInfo>&& formatDescription, uint64_t trackId, CompletionHandler<void(Ref<TrackInfo>&&, uint64_t)>&& completionHandler)
+{
+    auto updateFormatDescriptionOperation = UpdateFormatDescriptionOperation { WTFMove(formatDescription), trackId, WTFMove(completionHandler) };
```

Comment:
```suggestion
    UpdateFormatDescriptionOperation updateFormatDescriptionOperation { WTFMove(formatDescription), trackId, WTFMove(completionHandler) };
```
but really we should use a one liner with append.

Reply:
all this will soon go away